code/game/gamemodes/blob/powers.dm

```diff
@@ -276,7 +273,8 @@
 	last_attack = world.time
 	OB.expand(T, 0, blob_reagent_datum.color)
 	for(var/mob/living/L in T)
-		blob_reagent_datum.reaction_mob(L, TOUCH)
+		if(!prob(L.run_armor_check(pick("chest","head"), "bio")))
```

Comment:
Has to be this way, the large majority of blob chems will fuck you up the same degree no matter how much is actually injected into the mob, reductions are pointless.